### PR TITLE
Bug/sc 28033/add link to perform search on card

### PR DESCRIPTION
**Pull request by ksmartin301**
Added "Search on CARD" button for search queries to be directed to CARD website.

### CODE REVIEW

**Review by mfranz1 — CHANGES_REQUESTED** [review]
This also has changes from your editorial review process updates. These should be two different branches

File: src/app/admin/components/add-evaluator/components/object-dropdown/object-dropdown.component.ts [hunk 1]
```diff
@@ -106,6 +106,7 @@ export class ObjectDropdownComponent implements OnInit, OnDestroy {
     this.showDropdown = false;
   }
 
+
```
Comment (mfranz1) [inline src/app/admin/components/add-evaluator/components/object-dropdown/object-dropdown.component.ts:109]:
```suggestion
```
Remove extra spacing

File: src/app/cube/browse/browse.component.html [hunk 2]
```diff
@@ -17,6 +17,7 @@
       <div class="column-title" id="column-title">
         <span tabindex="0" id="results">{{ copy.RESULTS }} ({{ this.totalLearningObjects ? this.totalLearningObjects : '0' }})
           <a class="clear-search" id="clear-search" (activate)="clearSearch()" *ngIf="query?.text !== '' || query.standardOutcomes.length">{{ copy.CLEARSEARCH }} </a>
+          <a class="search-CARD" id="search-CARD" href="https://caeresource.directory/home/browse?q={{ query?.text }}" target=_blank *ngIf="query?.text !== '' || query.standardOutcomes.length">Search with CARD</a>
```
Comment (mfranz1) [inline src/app/cube/browse/browse.component.html:20]:
```suggestion
          <a class="search-CARD" id="search-CARD" href="https://caeresource.directory/home/browse?q={{ query?.text }}" target=_blank *ngIf="query?.text !== '' ">Search with CARD</a>
```

**Review by mfranz1 — CHANGES_REQUESTED** [review]

**Comment by katieblomquist** [comment]
Could you add screenshots of what the button on the page looks like as well as the resulting behavior?

**Review by Cwagne17 — APPROVED** [review]

**Review by bereketsega — APPROVED** [review]

**Comment by ksmartin301** [comment]
> Could you add screenshots of what the button on the page looks like as well as the resulting behavior?

Link to CARD
Search Page Results

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by mfranz1 — COMMENTED** [review]
This is looking good, just curious, did Paige or anyone give guidance on the placement of the button? I wouldn't mind meeting to discuss this

**Review by ksmartin301 — COMMENTED** [review]
moved CARD search to the right side
complie complete
end page

**Review by mfranz1 — COMMENTED** [review]
Perfect! One last thing, can you run `npm version minor` and push up that version patch? Then this will be good to deploy!

**Review by mfranz1 — APPROVED** [review]

**Review by shanioffe — APPROVED** [review]

**Pull request merged**